### PR TITLE
awk, chmod, chown: add Chinese translation

### PR DESCRIPTION
**Pull request by HugueLiu**
- **Version of the command being documented (if known):**  

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.zh/common/chown.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# chown
+
+> 修改用户和用户组对文件或目录的所有权。
+> 更多信息：<https://www.gnu.org/software/coreutils/chown>.
+
+- 修改文件或目录的所有者：
+
+`chown {{用户}}} {{文件或目录路径}}`
```
Comment (blueskyson) [inline pages.zh/common/chown.md:8]: Refer to [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md).

```suggestion
`chown {{用户}}} {{路径/到/文件或目录}}`
```

File: pages.zh/common/chown.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- 修改文件或目录的所有者及所属组：
+
+`chown {{用户}}:{{用户组}} {{文件或目录路径}}`
```
Comment (blueskyson) [inline pages.zh/common/chown.md:12]:
```suggestion
`chown {{用户}}:{{用户组}} {{路径/到/文件或目录}}`
```

File: pages.zh/common/chown.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- 递归修改目录及其子目录和文件的所有者：
+
+`chown -R {{用户}} {{目录路径}}`
```
Comment (blueskyson) [inline pages.zh/common/chown.md:16]:
```suggestion
`chown -R {{用户}} {{路径/到/目录}}`
```

File: pages.zh/common/chown.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- 修改符号链接的所有者：
+
+`chown -h {{用户}} {{符号链接路径}}`
```
Comment (blueskyson) [inline pages.zh/common/chown.md:20]:
```suggestion
`chown -h {{用户}} {{路径/到/符号链接}}`
```

File: pages.zh/common/chown.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- 修改文件或目录的所有者与参考文件相同：
+
+`chown --reference={{参考文件路径}} {{文件或目录路径}}`
```
Comment (blueskyson) [inline pages.zh/common/chown.md:24]:
```suggestion
`chown --reference={{路径/到/参考文件}} {{路径/到/文件或目录}}`
```

Comment (HugueLiu) [reply pages.zh/common/chown.md]: Thanks for your review, the translations have been fixed.

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**